vault/resource_database_secret_backend_connection.go

```diff
@@ -148,6 +148,10 @@ func (i *dbEngine) Name() string {
 	return i.name
```

Comment:
These aren't part of your changes, but I'm wondering if we need both `dbEngine.Name()` and `dbEngine.String()`. Looks like they are duplicate functions

Reply:
Yeah, `String()` should probably return `Name()`. `String()` implements the `Stringer` interface